scipy/special/tests/test_mpmath.py

```diff
@@ -111,6 +112,7 @@ def test_hyp2f1_strange_points():
         (2, -1, -1, 0.7),  # expected: 2.4
```

Comment:
As a final change, I think we can remove the xfail mark above.

Reply:
Aside from the MSVC failures (which I can fix later), the `hpy2f1` test didn't pass with the additional points. I can revert changes and then unmark the test as xfail, but there's probably an additional failure somewhere that will need a whole new round of investigation :(.